fsys_test.go

```diff
@@ -33,9 +33,18 @@ func TestMain(m *testing.M) {
 				if err := x.Start(); err != nil {
 					log.Fatalf("failed to execute Xvfb: %v", err)
 				}
-				// Give Xvfb some time to start up.
-				// 3 seconds is default for xvfb-run.
-				time.Sleep(3 * time.Second)
+				// Wait for Xvfb to start up.
+				for i := 0; i < 60*60; i++ {
```

Comment:
This is an hour if something goes wrong?

Reply:
Maybe it should be shorter.

Reply:
Yeah, one hour, although public repo jobs can't take longer than 50 minutes: https://docs.travis-ci.com/user/customizing-the-build/#build-timeouts

I don't know how Travis schedules jobs. I was kind of assuming jobs can be preempted for higher priority jobs. I can decrease the time. 5 minutes maybe?

Reply:
5 minutes sounds good to me. If it's not going to start in 5 minutes, we have a problem.

Reply:
I've changed it to 5 seconds.